cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -298,7 +298,7 @@ To put a value into a token, first obtain access to the token from the workflow
 and then set a value for a specific key. 
 
 In the case of a MapReduce program, the program's Mapper and Reducer classes need to
-implement ``ProgramLifecycle<MapReduceContext>``. After doing so, they can access the
+implement ``ProgramLifecycle<MapReduceTaskContext>``. After doing so, they can access the
```

Comment:
It's an API change that happened previously (around 3.4.0). It was backwards-compatible, though.
What's missing, do you think?